scvi/model/_condscvi.py

```diff
@@ -57,7 +58,7 @@ def __init__(
         n_hidden: int = 128,
         n_latent: int = 5,
         n_layers: int = 2,
-        dropout_rate: float = 0.1,
+        dropout_rate: float = 0.05,
```

Comment:
is that even used anywhere? see my other discussions of this param. Consider either using it or removing it in all the objects.